modules/universal/monitor/vars.tf

```
@@ -68,5 +68,10 @@ variable "targets" {
 
 variable "log_retention_period_days" {
   default     = 30
-  description = "Set the retention period of the aggregated log to the monitor server"
+  description = "Set the retention period of the aggregated log to the monitor host"
+}
+
+variable "log_archive_storage_info" {
+  default     = ""
+  description = "A info to set archive storage of the aggregated log to the monitor host"
```

```suggestion
  description = "An info to set archive storage of the accumulated logs in the monitor host"
```

It is not related to this PR but I think all the descriptions are a bit inconsistent in the sense that some start with a noun describing what it is and others start with a verb describing what it does.

---

Fixed in ff60449.